sdk/script/acquisition-sdk.ts

```diff
@@ -169,7 +172,17 @@ export class AcquisitionManager {
                     return;
             }
         }
-
+        
+        if (fromLabelOrAppVersion) {
```

Comment:
I'm not sure about `undefined`, but `null` would indeed create the property. Sure the server ignores it if it is falsey, but if we can save the network overhead of sending few bytes from the client side I think it makes sense to do it.